.github/workflows/ci.yml

```diff
@@ -165,10 +165,6 @@ jobs:
             env: { STACKLESS: true, PY: 3 }
             extra_hash: "-stackless"
           # Pypy
-          - os: ubuntu-20.04
-            python-version: pypy-2.7
-            backend: c
-            env: { NO_CYTHON_COMPILE: 1 }
```

Comment:
Stopping to test in PyPy2 basically means that we remove support for it already in Cython 3.0. Is that intended? We'll remove all Py2 support in Cython 3.1, whereas Cython 3.0 rather stands as it is now.

If we decide to remove PyPy2 support right now, there's probably more to rip out.

Reply:
The CI has not been testing PyPy2 since the [check is wrong](https://github.com/cython/cython/actions/runs/5708344400/job/15465930990) (note how neither "Setup Python" nor "Setup python 2" are run). So this is more of a admission of failure than a deprecation. With that, I can restore this and make sure the CI run will use PyPy2 if you prefer, although I view any effort to deal with PyPy2 + Cython as a time sink.

Reply:
I changed the CI action check fairly recently, so it probably worked before. Removing it entirely now seems a bit of a step.

But I won't argue with you about PyPy support. :) If you say that PyPy2 isn't used enough (at least together with Cython), then I agree that there's not much use in keeping it alive.

Reply:
Just to be sure, if you re-enable this, what happens? Does it fail? Does it still not use PyPy2? Is it actual work to get this running again?

Reply:
Restored the run and added logic to use pypy-2.7 in it

Reply:
BTW, this discussion got me to [ask on Hacker New](https://news.ycombinator.com/item?id=36940871) if anyone is using PyPy at all, and the answers so far are not that encouraging (although it has only been a few hours).